docs/source/tutorials/multiannotator.ipynb

```diff
@@ -0,0 +1,555 @@
+    "\n",
+    "- Obtain consensus labels of multiannotator data using majority vote\n",
+    "- Train a model on the majority vote consensus labels to compute out-of-sample predicted probabilites\n",
+    "- Use cleanlab's `multiannotator.get_label_quality_multiannotator` function to get improved consensus labels\n",
+    "- View other information about your multiannotator dataset, such as consensus and annotator quality scores, agreement scores, detailed label quality scores and more"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e6a48d31",
+   "metadata": {},
+   "source": [
+    "## 1. Install and import required dependencies"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6c6e5b15",
+   "metadata": {},
+   "source": [
+    "You can use `pip` to install all packages required for this tutorial as follows:\n",
+    "\n",
+    "```ipython3\n",
+    "!pip install sklearn\n",
+    "!pip install cleanlab\n",
+    "\n",
+    "# Make sure to install the version corresponding to this tutorial\n",
+    "# E.g. if viewing master branch documentation:\n",
+    "#     !pip install git+https://github.com/cleanlab/cleanlab.git\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a3ddc95f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Package installation (hidden on docs website).\n",
+    "dependencies = [\"cleanlab\", \"sklearn\"]\n",
+    "\n",
+    "if \"google.colab\" in str(get_ipython()):  # Check if it's running in Google Colab\n",
+    "    %pip install cleanlab  # for colab\n",
+    "    cmd = ' '.join([dep for dep in dependencies if dep != \"cleanlab\"])\n",
+    "    %pip install $cmd\n",
+    "else:\n",
+    "    missing_dependencies = []\n",
+    "    for dependency in dependencies:\n",
+    "        try:\n",
+    "            __import__(dependency)\n",
+    "        except ImportError:\n",
+    "            missing_dependencies.append(dependency)\n",
+    "\n",
+    "    if len(missing_dependencies) > 0:\n",
+    "        print(\"Missing required dependencies:\")\n",
+    "        print(*missing_dependencies, sep=\", \")\n",
+    "        print(\"\\nPlease install them before running the rest of this notebook.\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dd0148e6",
+   "metadata": {},
+   "source": [
+    "Let’s import some of the packages needed throughout this tutorial."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c4efd119",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "\n",
+    "from cleanlab.multiannotator import get_label_quality_multiannotator, get_majority_vote_label"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "345b6678",
+   "metadata": {},
+   "source": [
+    "## 2. Create the data (can skip these details)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "69b5ddaa",
+   "metadata": {},
+   "source": [
```

Comment:
Note the code in your markdown cell Pulldown  does not exactly match the code in the hidden python cell below.  In particular the hidden python cell has some extra code:


```
X = data_dict["X_train"]
multiannotator_labels = data_dict["multiannotator_labels"]
true_labels = data_dict["true_labels_train"]
```

Reply:
I'd prefer all that code and the execution of:
```
data_dict = make_data()
```
to be an entirely separate python cell (not hidden).  So the only thing that should be in this markdown pulldown cell and in the hidden cell would be the definition of the function, not the usage of the function to define global variables used elsewhere in the notebook.

Reply:
Moved the function call and subsequent lines to a non-hidden cell.

Reply:
I believe I also followed the format of other tutorials, so some of the ``make_data()`` calls in the existing tutorials might also be hidden. Can open a PR in the future to fix this